### DISCUSSION TITLE
Output general.php config settings in CP

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#9444] [Ideas]
> Created by: Stephen McIver (spm999@gmail.com) on 2015/04/13 23:58:09 +0000
> Votes at time of UserVoice import: 2

-----------------------------------

When a config setting is overriding a setting in the CP, you see a message like this in the CP:

"This is being overridden by the siteName config setting."

I think it'd be useful if the actual value of the setting from the config was outputted too, instead of having to open general.php to see it.

i.e. something like this (but with the actual value highlighted in a different colour or in italics or something, which I couldn’t do with the web inspector):

https://www.dropbox.com/s/5it803yuw9mr49p/Screenshot%202015-08-10%2019.05.33.png?dl=0

**Comment by AugustMiller** [comment]
Now that we are providing [environment-level overrides](https://craftcms.com/docs/5.x/configure.html#environment-overrides) for many `general.php` settings, this feels more essential.

We may not be able to report exactly the stage at which a value was set (config array or environment variable?), but we could at least let the user know what value is being used.

**Reply by thasmo** [reply]
I like that. 😏
